debian/changelog

```diff
@@ -1,3 +1,27 @@
+ostree (2017.7-1-endless1) unstable; urgency=medium
```

Comment:
OK, I’ll try `2017.7-1endless1`. One of these days I will remember the syntax for Debian package versioning.